src/foldermodel.cpp

```diff
@@ -366,11 +366,15 @@ QMimeData* FolderModel::mimeData(const QModelIndexList& indexes) const {
   for(const auto &index : indexes) {
     FolderModelItem* item = itemFromIndex(index);
     if(item) {
-      FmPath* path = fm_file_info_get_path(item->info);
-      char* uri = fm_path_to_uri(path);
-      urilist.append(uri);
-      urilist.append('\n');
-      g_free(uri);
+      if(item->info) {
```

Comment:
Yes, I know. I just wanted to emphasize that `item->info` may be null theoretically (I still don't know if that's possible in practice),